src/utils.js

```diff
@@ -28,7 +12,7 @@ export function replaceVars(text, vars = {}) {
   Object.keys(vars).forEach(key => {
     let val = vars[key];
 
-    text = text.replace(new RegExp(`{{${key}}}`, 'g'), val);
+    text = text.replace(new RegExp(`{{${key}}}`, 'g'), val); // Todo use https://github.com/tc39/proposal-string-replaceall in the future
```

Comment:
String.prototype.replaceAll should be at least twice faster, this quick polyfill shows it https://repl.it/@caub/replaceVars